src/main/io/osd.c

```diff
@@ -436,6 +436,23 @@ static void osdDrawSingleElement(uint8_t item)
             break;
         }
 
+    case OSD_REMAINING_TIME_ESTIMATE:
+        {
```

Comment:
There is already a function to format a time string which should be used instead of duplicating it's functionality here.

Also you have some broken indentation.